### PR TITLE
Add queue command User Guide

### PR DESCRIPTION
**Pull request by iamjackslayer**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -208,6 +208,36 @@ Selects the 2nd patient in the clinic record.
...
+Assigns the patient based on the index number used in the displayed patient list. +
+Format: `enqueuepatient INDEX`
+
+****
+* Assigns the patient into the queue (first in first out manner)
+* The index refers to the index number shown in the displayed patient list.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
+Examples:
```
Comment (arsalanc-v2) [inline docs/UserGuide.adoc]:
```suggestion
The command can also be used in combination with other commands.
Examples:
```

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -208,6 +208,36 @@ Selects the 2nd patient in the clinic record.
...
+
+* `list` +
+`enqueuepatient 7` +
+Selects the 7th patient in the displayed list of patients resulted from the `list` command
```
Comment (arsalanc-v2) [inline docs/UserGuide.adoc]:
```suggestion
Selects the 7th patient in the displayed list of patients resulting from the `list` command.
```

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -208,6 +208,36 @@ Selects the 2nd patient in the clinic record.
...
+
+* `find Logan` +
+`enqueuepatient 1` +
+Selects the 1st patient in the displayed list of patients whose name contain *Logan* (case insensitive) resulted from the `find Logan` command
```
Comment (arsalanc-v2) [inline docs/UserGuide.adoc]:
```suggestion
Selects the 1st patient in the displayed list of patients whose name contains *Logan* (case insensitive) resulting from the `find Logan` command.
```

Comment (iamjackslayer) [reply docs/UserGuide.adoc]: Noted with thanks

**Pull request closed without merging**